src/Data/RFC3339String.purs

```diff
@@ -39,6 +48,17 @@ trim (RFC3339String s) =
 fromDateTime :: DateTime -> RFC3339String
 fromDateTime = trim <<< RFC3339String <<< format iso8601Format
 
+-- | Reads the locale, returning GMT (+0000) if not present.
+toLocale :: RFC3339String -> Locale
+toLocale (RFC3339String s) = Locale Nothing $ fromMaybe zero $ unsafePartial $ do
+  re <- hush $ RE.regex "([-|\\+])(\\d\\d):?(\\d\\d)$" RE.noFlags
+  [_, sign, hrs, mins] <- sequence =<< RE.match re s
+  let readNum = map toNumber <<< fromString
+  hrs' <- readNum hrs
+  mins' <- readNum mins
+  let offset  = convertDuration (Hours hrs') + Minutes mins'
+  pure $ (if sign=="-" then negate else id) offset
```

Comment:
Total nitpick on the spacing here. I think there's some extra before the `=` in `offset  =`, and I would normally surround binary operators like `==` with spaces to make it `sign == "-"`.

I could fix this afterwards though if you like.

Reply:
Fixed